content/blog/asyncapiathacktoberfest.md

```diff
@@ -0,0 +1,98 @@
+
+TODO: put video here
+
+### 2. Setup Your Environment And Create A First Pull Request
+
+Once you [install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) on your machine and get a [GitHub account](https://github.com/join), you need first to decide if you are here just for Hacktoberfest or longer, and make sure if your issue is easy and maybe you can complete it in the GitHub UI. 
+
+In case you are here just for the Hacktoberfest, and you picked easy issues that involve changes only to a single file, there is no need to install Git and complicate your life. GitHub UI enables you to [make changes to a single file online](https://docs.github.com/en/free-pro-team@latest/github/managing-files-in-a-repository/editing-files-in-your-repository).
+
+In case you:
+
+- want to stay with us longer,
+- you picked up an issue where you need to make changes to more than just one file,
+- you also need to run the project locally to check if it works
+
+Then follow [this](https://github.com/asyncapi/.github/blob/master/git-workflow.md) short instruction on how to fork the repository and set it up locally.
+
+Once you are ready with your changes, submit a pull request. Be nice and follow our [code of conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md) and make sure your pull request is [described properly](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md#conventional-commits).
+
+{{%youtube BsC5tu4M1rw%}}
+ 
+## Office Hours
+
+Do you feel overwhelmed? No need. You can do it. Just take this blog post seriously. 
+
+Trust me when I write that every pull request is crucial for us.
+Trust me when I write that we are a welcoming community.
+Don't be afraid that you will waste our time. If we would think about it this way, we would not even join the Hacktoberfest.
+
+Still not sure if you can make it? Don't worry. We want to host office hours throughout the event, 2x a week, 1h long, and different timezones. You can join whenever you want and ask us anything, or do pair programming with us. We start with the first meeting on [Tuesday 6th, 8AM UTC](https://calendar.google.com/calendar/u/0?cid=dGJyYmZxNGRlNWJjbmd0OG9rdmV2NGxzdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ) and then on the following days:
+
+- Tuesday 6th, 8AM UTC
+- Thursday 8th, 4PM UTC
+- Tuesday 13th, 8AM UTC
+- Thursday 15th, 4PM UTC
+- Tuesday 20th, 8AM UTC
+- Thursday 22nd, 4PM UTC
+- Tuesday 27th, 8AM UTC
+- Thursday 29th, 4PM UTC
+
+You can also join us in a more asynchronous discussion on [Slack](https://www.asyncapi.com/slack-invite/). For updates and latest news, the best is to follow our [Twitter account](https://twitter.com/AsyncAPISpec). 
+
+Enjoy the Hacktoberfest!
+
+## Blooper Reel
+
+Before you jump to your first contribution, have a look a the making of the videos. It was quite fun.
+
+{{%youtube anjcF2l0lGs%}}
+
```

Comment:
I'm feeling like a goodbye message or something is missing here. Maybe the last sentence to motivate people?

Reply:
I don't like motivation speeches, how about now?

Reply:
Fair enough! 👍